settings.gradle

```diff
@@ -4,7 +4,12 @@ include ':redwood-cli'
 include ':redwood-compose'
 include ':redwood-flexbox-engine'
 include ':redwood-gradle-plugin'
-include ':redwood-layout'
+include ':redwood-layout-api'
+include ':redwood-layout-schema'
+include ':redwood-layout-schema:compose'
+include ':redwood-layout-schema:layout-modifiers'
+include ':redwood-layout-schema:widget'
+include ':redwood-layout-view'
```

Comment:
Put this here because we'll need a `-composeui` version. I'm not sure it makes sense to ship a MPP artifact with Android views + iOS UIKit then have a separate Compose UI artifact, compared to just have three separate artifacts. Of course, the design of the existing widget and treehouse artifacts does that so maybe I'm wrong.

Reply:
I think I'd prefer separate `view`, `uikit`, and `composeui` artifacts as well since there will probably be consumers who are 100% compose UI and don't need the separate view-based implementations.